api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java

```diff
@@ -369,18 +369,22 @@ public Long findDomainCountByStandard(String domain, String term, Boolean standa
         : count;
   }
 
+  @Override
+  public List<CardCount> findEhrDomainCounts(String term) {
+    return findDomainCounts(
+        term,
+        true,
+        ImmutableList.of(
+            Domain.CONDITION,
+            Domain.DRUG,
+            Domain.MEASUREMENT,
+            Domain.OBSERVATION,
+            Domain.PROCEDURE));
```

Comment:
Sorry, missed this before. List needs to include Visit and Devices

Reply:
ok got it.

Reply:
no particular order, right?

Reply:
The order of the cards for the UI is in the domain_card table sort_order column.

Reply:
but for Universal search we may want to return them in order by highest count

Reply:
done!